### PR TITLE
[SERV-805] Display web contact field on institution pages

### PR DESCRIPTION
**Pull request by markmatney**
Also while I was at it, decided to specify `tel:` scheme for phone numbers. Because why not.

### CODE REVIEW

File: ucla-prrla/src/app/institution/institution.component.html [hunk 1]
```diff
@@ -25,9 +25,11 @@
                             <dt>Location: </dt>
                             <dd><strong>{{ memberInfo.prrla_member_location }}</strong></dd>
                             <dt>Phone: </dt>
-                            <dd><strong>{{ memberInfo.prrla_member_phone }}</strong></dd>
+                            <dd><strong><a href="tel:{{ memberInfo.prrla_member_phone }}">{{ memberInfo.prrla_member_phone }}</a></strong></dd>
                             <dt>Email: </dt>
                             <dd><strong><a href="mailto:{{ memberInfo.prrla_member_email }}">{{ memberInfo.prrla_member_email }}</a></strong></dd>
+                            <dt>Web contact: </dt>
+                            <dd><strong><a href="{{ memberInfo.prrla_member_web_contact }}">{{ memberInfo.prrla_member_web_contact }}</a></strong></dd>
```
Comment (DRickard) [inline ucla-prrla/src/app/institution/institution.component.html]: Maybe use a target value in the `href`, so the PRRLA page is still open/usable?

Comment (markmatney) [reply ucla-prrla/src/app/institution/institution.component.html]: Since it's such an easy changes: sure.

Comment (markmatney) [reply ucla-prrla/src/app/institution/institution.component.html]: Oh, nvm, I see the Website anchor tag has a blank target. 👍🏻

Comment (markmatney) [reply ucla-prrla/src/app/institution/institution.component.html]: ✅

**Review by DRickard — APPROVED** [review]

**Review by ksclarke — APPROVED** [review]

**Pull request merged**